### PR TITLE
Add node2vec

### PR DESCRIPTION
**Pull request by cu8blank**
I would like to add the node2vec Python library to conda-forge.

### CODE REVIEW

File: recipes/node2vec/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - networkx==1.11
+    - numpy==1.11.2
+    - gensim==0.13.3
```
Comment (sodre) [inline recipes/node2vec/meta.yaml]: Please make sure network, numpy and gensim are really build time dependencies and not only runtime ones.

File: recipes/node2vec/LICENSE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+MIT License
```
Comment (sodre) [inline recipes/node2vec/LICENSE]: Thanks for packaging the license. But please open an issue [upstream](https://github.com/eliorc/node2vec) asking them to include the license in the source distro in pypi.

File: recipes/node2vec/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - node2vec
+
+about:
+  home: https://github.com/aditya-grover/node2vec
```
Comment (sodre) [inline recipes/node2vec/meta.yaml]: The home page according to the PyPi page is actually https://github.com/eliorc/node2vec. Please verify.

File: recipes/node2vec/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'The node2vec algorithm learns continuous representations for nodes in any (un)directed, (un)weighted graph.'
+
+  description: |
+    The node2vec algorithm learns continuous representations for nodes in any (un)directed, (un)weighted graph.
+  doc_url: http://snap.stanford.edu/node2vec/
+  dev_url: https://github.com/aditya-grover/node2vec
```
Comment (sodre) [inline recipes/node2vec/meta.yaml]: I think this should be https://github.com/eliorc/node2vec

File: recipes/node2vec/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+requirements:
+  build:
+    - python
+    - pip
+    - networkx==1.11
+    - numpy==1.11.2
+    - gensim==0.13.3
+  run:
+    - python
+    - networkx==1.11
```
Comment (sodre) [inline recipes/node2vec/meta.yaml]: I think the dependency list should actually come from [here](https://github.com/eliorc/node2vec/blob/2ff4bf481bae0a2120d66c350254dbd74137ab9e/setup.py#L12-L18).

File: recipes/node2vec/meta.yaml (hunk 1 above)
Comment (cu8blank) [reply recipes/node2vec/meta.yaml]: These are only runtime dependencies. I deleted them from the meta.yaml file.

File: recipes/node2vec/meta.yaml (hunk 3 above)
Comment (cu8blank) [reply recipes/node2vec/meta.yaml]: You're right. I'm sorry I missed that. I fixed this in the meta.yaml file.

File: recipes/node2vec/meta.yaml (hunk 4 above)
Comment (cu8blank) [reply recipes/node2vec/meta.yaml]: You're right. I'm sorry I missed that. I fixed this in the meta.yaml file.

**Comment by sodre** [comment]
Welcome to conda-forge and thanks for creating the node2vec conda package.

**Pull request merged**